sdk/storage/Azure.Storage.Common/src/Shared/Constants.cs

```diff
@@ -449,6 +449,16 @@ internal static class DataLake
             /// The Azure Storage error codes for Datalake Client.
             /// </summary>
             public const string AlreadyExists = "ContainerAlreadyExists";
+
+            /// <summary>
+            /// Default concurrent transfers count.
+            /// </summary>
+            public const int DefaultConcurrentTransfersCount = 5;
+
+            /// <summary>
+            /// Max upload bytes.
+            /// </summary>
+            public const int MaxUploadBytes = 100 * Constants.MB; // 100 MB
```

Comment:
can we name this DataLakeMaxUploadBytes? that's what this is for right?

Reply:
It's in the DataLake class, so when we refer to it, it will be Constants.DataLake.MaxUploadBytes, so we will know what its for